CONTRIBUTING.md

```diff
@@ -23,3 +23,27 @@ If you'd like to help make RSpec better, here are some ways you can contribute:
 If you need help getting started, check out the [DEVELOPMENT](DEVELOPMENT.md) file for steps that will get you up and running.
 
 Thanks for helping us make RSpec better!
+
+## `Small` issues
+
+These issues are the ones that we be believe are best suited for new
+contributors to get started on. They represent a potential meaningful
+contribution to the project that should not be too hard to pull off.
+## `Triage` issues
+
+These issues are ones that have been labelled by the maintainers that we
+believe do not currently have enough information to be reproduced the RSpec
+team. You can help us move triage issues into `ready to go` or `has reproduction
+case` (for feature requests or bugs) as a seriously meaningful contribution.
```

Comment:
The way this sentence is worded, it sounds like people can help us by changing the issue label. I think it would be better to state what is needed to get a triaged issue into one of these states.